README.md

```diff
@@ -254,6 +254,28 @@ yarn stats
 
 ### Settings
 
+#### Organization Configuration
+
+Organization configuration can be generated by running:
+`yarn add-org`
+
+Organization configuration is generated from the template present in `internals/generators/config.yml.hbs`.
+Default configuration is present in `internals/config/default.yml`. If the configuration file of a specific
+organization misses a piece of configuration then this default configuration is used to generate
+complete configuration.
+
+To remove the specific section of the configuration, you can use the `null` keyword to delete
```

Comment:
The part on removing sections can be kept here (with a dedicated heading) but I guess that if we go with the proposed changes it will have to be updated.